csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/NativeLibraryInclude.props

```diff
@@ -0,0 +1,171 @@
+<!--
+Add the native libraries from either a local build or a prebuilt native nuget package.
+
+This has to be imported by the test project with the actual target platform/frameworks to work correctly as the common
+test project only targets net8 and netstandard2.0.
+-->
+<Project>
+    <PropertyGroup>
+        <!-- build host system -->
+        <IsWindowsBuild Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">true</IsWindowsBuild>
+        <IsLinuxBuild Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' == 'true'">true</IsLinuxBuild>
+        <IsMacOSBuild Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' == 'true'">true</IsMacOSBuild>
+
+        <!-- set for MAUI targets -->
+        <IsWindowsTarget Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">true</IsWindowsTarget>
+        <IsAndroidTarget Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">true</IsAndroidTarget>
+        <IsIOSTarget Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">true</IsIOSTarget>
+        <IsMacCatalystTarget Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'">true</IsMacCatalystTarget>
+
+        <!--
+        Allow a pre-built ORT native nuget package (Microsoft.ML.OnnxRuntime.<version>.nupkg) to be used.
+
+        The test projects that include this file must be built from the command-line to enable using a prebuilt package.
+        Current test projects:
+            - Microsoft.ML.OnnxRuntime.Tests.NetCoreApp
+            - Microsoft.ML.OnnxRuntime.Tests.MAUI
+
+        If running from the repo root the below is an example command.
+            Note that '==' represents a double '-' which isn't allowed in an XML comment
+            Properties can also be set via environment variables.
+
+        dotnet build csharp\test\Microsoft.ML.OnnxRuntime.Tests.MAUI\Microsoft.ML.OnnxRuntime.Tests.MAUI.csproj
+            ==property:UsePrebuiltNativePackage=true
+            ==property:CurrentOnnxRuntimeVersion=1.19.2
+            ==source <path containing the Microsoft.ML.OnnxRuntime.<version>.nupkg>
+            ==source https://api.nuget.org/v3/index.json
+
+        The <version> of the nupkg must match the value provided in CurrentOnnxRuntimeVersion.
+
+        The "==source" args are not required if a released Microsoft.ML.OnnxRuntime package is being used.
+        If using a previous release you must ensure it is compatible with the entries in NativeMethods.shared.cs.
+        If new bindings have been added recently you will get error when those are initialized if the native code is out
+        of date and does not match.
```

Comment:
Should the info about NativeMethods also be added to the tests readme -- especially since I think it might be unlikely a user will read the NativeLibraryInclude.props file

Reply:
Added